packages/lab/src/avatar/Avatar.tsx

```diff
@@ -1,74 +1,52 @@
-import { makePrefixer, useId } from "@salt-ds/core";
+import { FlexLayout, makePrefixer } from "@salt-ds/core";
+import { UserIcon } from "@salt-ds/icons";
 import { clsx } from "clsx";
-import { forwardRef, HTMLAttributes, ImgHTMLAttributes } from "react";
-import { classBase } from "./internal/constants";
-import { DefaultAvatar } from "./internal/DefaultAvatar";
-import { useLoaded } from "./internal/useLoaded";
+import { forwardRef, HTMLAttributes, ReactNode } from "react";
 
 import "./Avatar.css";
 
+const classBase = "saltAvatar";
+
 export interface AvatarProps extends HTMLAttributes<HTMLDivElement> {
-  alt?: string;
-  id?: string;
-  imgProps?: ImgHTMLAttributes<HTMLImageElement>;
-  size?: "small" | "medium" | "large";
-  sizes?: string;
-  src?: string;
-  srcSet?: string;
-  title?: string;
+  children?: ReactNode | string;
+  size?: number;
+  fallbackIcon?: ReactNode;
 }
 
 const withBaseName = makePrefixer(classBase);
+const DEFAULT_AVATAR_SIZE = 2; // medium
 
 export const Avatar = forwardRef<HTMLDivElement, AvatarProps>(function Avatar(
   {
-    alt = "",
     className,
     children: childrenProp,
-    id: idProp,
-    size = "medium",
-    src,
-    srcSet,
-    title = "user",
-    imgProps,
-    sizes,
+    size = DEFAULT_AVATAR_SIZE,
+    style: styleProp,
+    fallbackIcon = <UserIcon />,
```

Comment:
What is the use case for fallbackIcon? I think it would make sense to build an example using it. In my mind, I'm trying to figure out why this wouldn't just be a default to children.

Reply:
I think I agree. The idea was previously that you provide the Avatar all the information (imgSrc, initial data and fallbackIcon) and it would sort out what gets displayed in order of preference. But having moved the responsibility for the image loading to the consumer it can probably all be children.

Reply:
Related question is we want to support both centered Icons e.g. the default, and also svgs which take up the full background circle of the Avatar (primarily for use in the Flag components).

Currently that's controlled by where it has the salt icon class. Are we happy with that logic?